extensions/wikia/PortableInfobox/tests/nodes/NodeGroupTest.php

```diff
@@ -48,11 +48,11 @@ public function groupNodeTestProvider() {
 			  [ 'elem1' => 1, 'elem2' => 2 ],
 			  [ 'value' =>
 					[
-						[ 'type' => 'data', 'isEmpty' => false, 'data' => [ 'label' => 'l1', 'value' => 1 ],
+						[ 'type' => 'data', 'isEmpty' => false, 'data' => [ 'label' => 'l1', 'value' => 1, 'collapse' => null ],
 						  'source' => [ 'elem1' ] ],
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "source" 18 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)